test_tortoise_interpreter.py

```diff
@@ -67,28 +61,18 @@ def test_sanitized_programs(program, expected):
 
 
 def test_navigate_command_functions():
-
-    COMMANDS = {'U': pen_up,'D': pen_down ,'P': pen_colour, 'N': move_north, 'E': move_east}
-    program = [['P', '3'], ['U'], ['N', '1'], ['U'], ['D'], ['E', '2'], ['D']]
+    def underscore(command, pen_state):
+        return '___', pen_state
+    def double_colon(command, pen_state):
+        return '::', pen_state
+    def hard_stop(commands, pen_state):
```

Comment:
I would like to see one of these handler functions use the arguments. Maybe the command should be `['H', '4']`, meaning it should print out four full stops. Maybe one could print out the pen state. 

However, this is much, much better than before :+1: